Makefile

```diff
@@ -413,6 +417,7 @@ microk8s: check-microk8s ## Build cilium-dev docker image and import to microk8s
 	$(QUIET)./contrib/scripts/microk8s-import.sh $(LOCAL_OPERATOR_IMAGE)
 
 precheck: logging-subsys-field ## Peform build precheck for the source code.
+	$(QUIET)$(MAKE) check-permissions
```

Comment:
Also, is this something we want to do for each build, or should this be restricted to CI only? What's the point of having contributors run this when running `make` for developing?

I think it should be enough to run it in CI, and catch files that have not been added with the right permissions at that time. It's easy enough to fix, even after creating the pull request.

Reply:
That is actually run only in CI, but moved it to separate target anyway, to make it more obvious.

Reply:
Oh sorry, thank you!